bundles/org.openhab.binding.tplinksmarthome/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -19,43 +19,60 @@
 		<category>Switch</category>
 	</channel-type>
 
+	<!-- Absolute Temperature Channel -->
+	<channel-type id="colorTemperatureAbs1" advanced="true">
```

Comment:
Would this be better of with a more descriptive name?

Reply:
I generalized it as the difference is both min and max color temperature for different devices, requiring different temp ranges. I didn't want to use the name of a specific device that has these ranges as there are multiple devices. The convention 1 and 2 is also used in the java code. The channel id is not directly visible to the user. The user refers to the `colorTemperatureAbs` channel name. But I couldn't come up with a better name. So if you have any ideas.